SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec

```diff
@@ -0,0 +1,50 @@
+Summary:        Package to deploy azurelinux-sysinfo service
+Name:           azurelinux-sysinfo
+Version:        %{azl}.0
+Release:        1%{?dist}
+License:        GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+Group:          System Environment/Base
+URL:            https://aka.ms/azurelinux
+Source0:        collect-sysinfo.py
+Source1:        sysinfo-schema-v1.json
+Source2:        azurelinux-sysinfo.service
+Requires:       systemd
+Requires:       python3-psutil
+
+%description
+Deploys a systemd service that collects system information related to the device, operating system, cloud-init, boot 
+time, resource utilization, installed packages, and SELinux mode. Collected information is written in JSON format to
+a log file on the user's system for easy access and analysis. The systemd service runs at boot time if installed during 
+image creation.
+
+%install
+# Copy collection python script to /usr/local/bin/
+mkdir -p %{buildroot}/usr/local/bin/
```

Comment:
typically distro provided packages shouldn't install to /usr/local. You can use /usr/bin or the appropriate macro (like bindir) here.